src/hotspot/share/utilities/ostream.hpp

```diff
@@ -70,20 +71,24 @@ class outputStream : public CHeapObjBase {
 
  public:
    // creation
-   outputStream();
-   outputStream(bool has_time_stamps);
+   outputStream(bool has_time_stamps = false);
 
    // indentation
    outputStream& indent();
    void inc() { _indentation++; };
    void dec() { _indentation--; };
    void inc(int n) { _indentation += n; };
    void dec(int n) { _indentation -= n; };
-   int  indentation() const    { return _indentation; }
-   void set_indentation(int i) { _indentation = i;    }
```

Comment:
Why have you deleted these?

Reply:
They were not needed, nobody uses them. Indentation level is only modified by outputStream itself and it accesses _indentation directly.

Reply:
`indention()` is being used in this PR: https://github.com/openjdk/jdk/pull/19482

Though I am querying the correctness/appropriateness of the logic there.